tests/lib/webbRelayer.ts

```diff
@@ -559,6 +565,12 @@ export interface RelayerMetricResponse {
   metrics: string;
 }
 
+export interface ResourceMetricResponse {
+  totalGasSpent: number,
+  totalFeeEarned: number,
+  accountBalance: number
+}
```

Comment:
These are strings, no?
```suggestion
export interface ResourceMetricResponse {
  totalGasSpent: string,
  totalFeeEarned: string,
  accountBalance: string
}
```